modules/c++/net.ssl/wscript

```diff
@@ -2,8 +2,119 @@ NAME            = 'net.ssl'
+                }
+                '''
+            sslFragment100 = '''
+                #include "openssl/ssl.h"
+                int main()
+                {
+                    SSL_library_init();
+                    SSL_load_error_strings();
+
+                    const SSL_METHOD *method = SSLv23_client_method();
+                    SSL_CTX* ctx = SSL_CTX_new(method);
+                    return 0;
+                }
+                '''
+
+            if sslHome:
+                if conf.check(lib=sslLib, uselib_store=sslUselib,
+                              header_name=sslHeader,
+                              function_name=sslFunction,
+                              fragment=sslFragment098,
+                              libpath=os.path.join(sslHome, 'lib'),
+                              includes=os.path.join(sslHome, 'include'),
+                              msg='Checking for version 0.9.8 of ssl library',
+                              okmsg=sslHome,
+                              define_name=sslOldDef,
+                              mandatory=False):
+                    conf.define(sslOldDef, True)
+                else:
+                    conf.check(lib=sslLib, uselib_store=sslUselib,
+                               header_name=sslHeader,
+                               function_name=sslFunction,
+                               fragment=sslFragment100,
+                               libpath=os.path.join(sslHome, 'lib'),
+                               includes=os.path.join(sslHome, 'include'),
+                               msg='Checking for version 1.0.0 of ssl library',
+                               okmsg=sslHome,
+                               mandatory=True)
+                conf.define(sslDef, True)
+            elif conf.check(lib=sslLib, uselib_store=sslUselib,
+                            header_name=sslHeader,
+                            function_name=sslFunction,
+                            fragment=sslFragment098,
+                            msg='Checking for version 0.9.8 of ssl library',
+                            mandatory=False):
+                conf.define(sslDef, True)
+                conf.define(sslOldDef, True)
+            elif conf.check(lib=sslLib, uselib_store=sslUselib,
+                            header_name=sslHeader,
+                            function_name=sslFunction,
+                            fragment=sslFragment100,
+                            msg='Checking for version 1.0.0 of ssl library',
+                            mandatory=False):
+                conf.define(sslDef, True)
+
+            # Update library information
+            if 'LIB_SSL' in conf.env:
+                self.uselib = conf.env['LIB_SSL']
+            if 'LIBPATH_SSL' in conf.env:
+                self.libpath = conf.env['LIBPATH_SSL']
+            if 'INCLUDES_SSL' in conf.env:
+                self.includes = conf.env['INCLUDES_SSL']
+
+        def updateConf(self, conf):
```

Comment:
I see.  Yeah this is a bummer but I'm not sure that there's a better way to do it...